### ISSUE TITLE
Icon Request: fa-tiki

### ISSUE THREAD

**Issue by gezzzan** [FortAwesome/Font-Awesome#7973]
Hello!

I am a long time contributor to the Tiki project (https://tiki.org, https://github.com/tikiorg/tiki) and I would like to request a new icon for the Tiki community.

Since version 14.x Tiki uses font awesome as its default iconset and it would be nice to have a brand icon included in the package. You can find the official tiki brand icon here: https://branding.tiki.org/Logo

I would gladly provide financial support to make this happen, please contact me about it

many thanks for your great work,
cheers,
gezza

**Comment by amette** [comment]
Yes, a Tiki brand icon would be a great addition that I would be very thankful for, too!

thanks
amette

**Comment by WizarDave** [comment]
A Tiki brand icon would be great.
They are using Font-Awesome as their default icon set now.
Thanks in advance for your consideration.
WizarDave

**Comment by bsfez** [comment]
Tiki use Font-Awesome extensively !
Thanks for adding a Tiki icon in Font-Awesome.

**Comment by torstenfabricius** [comment]
Hello dear folks,

back in 2013 we desided to move from a unique CSS design system to the Bootstrap Framework and in 2014 it appeared that our community desided to use Fontawesome as default.

Fontawesome is great and so is our software - a huge project and one of the best versatile content management systems for collaborative and special use cases especially when a tight integration of several fetures is demanded.

We are a large and active community, taking part in free open source development since far more then 10 years. I am convinced that we deserve an icon.

Whenever you consider to give us the honor of a "fa-tiki", please contact us via our website or here and we support you with any svg grafics or whatever you need at any time.

Thank you for reading and thank you for your great work!

Torsten
Tiki Community Team

**Comment by luciash** [comment]
+1! Thanks!

**Comment by johncourtland** [comment]
"fa-tiki" bring it on! It's about time! Thank you for your consideration!

**Comment by ricks99** [comment]
+1 for a tiki.org icon.

**Comment by bsfez** [comment]
It’s been a while, no news ?

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No biggie.
